lib/datasets/wine.rb

```diff
@@ -0,0 +1,64 @@
+require 'csv'
+
+require_relative 'dataset'
+
+module Datasets
+  class Wine < Dataset
+    Record = Struct.new(:wine_class,
```

Comment:
Did you choose `wine_class` instead of `class` to avoid conflict with `Kernel#class`?
We already use `class` in `Iris`. Should we change the `class` in `Iris`?

Reply:
fixed